### PR TITLE
Update Yarn.lock

### PR DESCRIPTION
**Pull request by samuelfreiberg**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Yarn.lock file was out-of-date. Leading to errors in the CI Pipeline during yarn install. Updating yarn.lock here

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by ayush547 — APPROVED** [review]

**Review by Saadnajmi — APPROVED** [review]

**Comment by Saadnajmi** [comment]
I'm gonna keep commenting this everyone I see it:

"More reason for us to move to yarn 3 to said ever getting this error in the first place"

**Review by lyzhan7 — APPROVED** [review]

**Comment by samuelfreiberg** [comment]
> I'm gonna keep commenting this everyone I see it:
> 
> "More reason for us to move to yarn 3 to said ever getting this error in the first place"

Just for my own knowledge - how would yarn 3 prevent this?

File: yarn.lock [hunk 1]
```diff
@@ -1470,24 +1470,6 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.42.0.tgz#6b7ca87ed30a92634e72cbe8fc1280fd21db50b8"
   integrity sha512-hBPCikSDJ3Rgw+mAbTgnNB83dn3SeSKPo4XNli22tbV08fN8P5mEaEg8Uw0cbVMN3D138ykNeAaK/lZjO9KIqA==
 
-"@fluentui-react-native/framework@0.8.31":
```
Comment (mischreiber) [inline yarn.lock:1473]: I think I already accidentally fixed this in #2554?

https://github.com/microsoft/fluentui-react-native/pull/2554/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de

Comment (samuelfreiberg) [reply yarn.lock:1473]: Looks like you did!

**Pull request closed without merging**